src/components/CardListBox/CardFolderList/CardFolderList.tsx

```diff
@@ -0,0 +1,33 @@
+import Folder from "./Folder/Folder";
+import styles from "./CardFolderList.module.scss";
+import { UserFolderProps } from "@/constants/index.types";
+
+interface CardFolderListProps {
+  folders?: UserFolderProps[];
+  handleFolderClick: (folder: UserFolderProps) => void;
+  handleTotalBtnClick: () => void;
```

Comment:
컴포넌트가 props로 이벤트 핸들러 함수를 받는 경우에는 on- prefix를 사용해주는게 일반적인 컨벤션입니다!
```suggestion
  onFolderClick: (folder: UserFolderProps) => void;
  onTotalBtnClick: () => void;
```